upgraderesponder/service.go

```diff
@@ -263,17 +263,21 @@ func (s *Server) GenerateCheckUpgradeResponse(request *CheckUpgradeRequest) (*Ch
 
 	// Only supports `latest` label for now
 	//latestVer, version, err := s.getParsedVersionWithTag(VersionTagLatest)
```

Comment:
We can remove those (with the comment) if it's no longer the case.

Reply:
Thank you. Fixed